pontoon/errorchecks/compare_locales.py

```diff
@@ -0,0 +1,128 @@
+import os
+from collections import (
+    namedtuple,
+)
+from compare_locales.checks import getChecker
+from compare_locales.paths import File
+from fluent.syntax import FluentParser
+
+
+# Because we can't pass the context to all entities passed to compare locales,
+# we have to create our equivalents of compare-locale's internal classes.
+ComparePropertiesEntity = namedtuple(
+    'ComparePropertiesEntity',
+    (
+        'key',
+        'val',
+
+        # We'll remove these fields at some point, currently they're required because of the current
+        # implementation property files in compare-locales.
+        'raw_val',
+        'pre_comment',
+    )
+)
+CompareDTDEntity = namedtuple(
+    'CompareDTDEntity',
+    (
+        'key',
+        'val',
+        'pre_comment',
+        'all',
+    )
+)
+CompareFluentEntity = namedtuple(
+    'FluentCompareEntity',
+    (
+        'entry',
+    )
+)
+
+ftl_parser = FluentParser()
+
+
+DTD_ENTITY_TMPL = '<!ENTITY %s \"%s\">'
+
+
+def cast_to_compare_locales(resource_ext, entity, translation):
+    """
+    Cast a Pontoon's translation object into Entities supported by `compare-locales`.
+    """
+    if resource_ext == '.properties':
+        if entity.plural_form:
```

Comment:
AFAIK, we don't support the funny plural form in .properties, but only gettext plurals with this db model?

Reply:
Correct.